src/server/scripts/Kalimdor/CavernsOfTime/BattleForMountHyjal/boss_archimonde.cpp

```diff
@@ -284,23 +290,28 @@ class boss_archimonde : public CreatureScript
 
             Enraged = false;
             HasProtected = false;
+            summons.DespawnAll();
+            Creature* WorldtreeTraget = reinterpret_cast<Creature*>(me->SummonCreature(WORLDTREE_CHANNEL_TARGET, 5503.713f, -3523.436f, 1608.781f, 0.0f, TEMPSUMMON_TIMED_DESPAWN, 360000s));
+            DoCast(WorldtreeTraget, SPELL_DRAIN_WORLD_TREE);
+            WorldtreeTraget->AI()->DoCast(me, SPELL_DRAIN_WORLD_TREE_TRIGGERED);
```

Comment:
That is handled by spell script in this file, doesn't work?

Reply:
The orignal script for SPELL_DRAIN_WORLD_TREE is broken.Archimode just stand and do nothing after spawned

Reply:
That is only part of it

Reply:
That is because he is invisible when you test it, all of that works correctly not inside instance(without any changes)
There is a way to fix it but it requires more changes in instance script. This PR is already too big to review it easy

Reply:
I found position and movement is wrong of original world tree channel trigger.The trigger keep walking and interrupt ths channel spell.Updated code